neurow/lib/neurow/public_api/endpoint.ex

```diff
@@ -225,6 +225,8 @@ defmodule Neurow.PublicApi.Endpoint do
             jwt_exp_s
           )
 
+        :erlang.garbage_collect()
```

Comment:
What is the perf impact of this?

Reply:
@bpaquet, @papey  

This is expected to be very quick because it only applied to the current process. Locally, most of the manual GC take about 6 microseconds, with a very small sets of outliers up to 20 microseconds.

Also for more context, Phoenix channels are hibernated after 15s of inactivity. Hibernating internally trigger a GC and does not seem to be a problem. So I think triggering process garbage collections is not a problem.

> One open question if the overall CPU impact on this operation distributed 10k processes.
Yes this is my main concern actually - The goal is to support about 100k per with a server with 10go RAM

I added a debounce to the manual garbage collection:
- A first GC is triggered after the SSE initialization, just after sending the history. This should be the more efficient one, because it should free all temporary resources used by cowboy and plug to initiate the HTTP connection.
- Then in the SSE passive loop, manual GC are performed max every 1 minutes when the passive loop run. (The PR does not change the frequency of the passive loop).